docs/source/getting_started/getting_started.ipynb

```diff
@@ -145,12 +145,13 @@
     "`~/my-private-bucket` is an S3 bucket with persistent storage, but accessible only to you and others in a shared workspace.\n",
```

Comment:
<p>Under the "My Root Folder",  next to</p><blockquote>Git is more likely to behave predictably here compared to other storage</blockquote><p>Jupyter Notebooks should be listed, along with anything that is edited often. Buckets are for long term storage, read-only data (it doesn't mean it's read-only but should be treated like it)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
What else is edited often? I can think of Python files (and like markdown and text files, but do we want to get that specific?)

Reply:
Most common examples is fine: code which include ipynb, md, .R, .py etc... including git repositories (which are mostly those formats)

Reply:
I added to this line because it seemed to fit better
Use this for code-related items, smaller data storage (such as Jupyter Notebooks, Python/ R/ Markdown files, etc.)